gordo_components/builder/build_model.py

```diff
@@ -125,6 +129,28 @@ def build_model(
     return model, metadata
 
 
+def _determine_offset(model: BaseEstimator, X: Union[np.ndarray, pd.DataFrame]) -> int:
+    """
+    Determine the model's offset. How much does the output of the model differ
+    from its input?
+
+    Parameters
+    ----------
+    model: sklearn.base.BaseEstimator
+        Trained model with either ``predict`` or ``transform`` method, preference
+        given to ``predict``
```

Comment:
Nitpick, but this could have a punctuation at the end, or remove the one below, depending on whether you think this is a complete sentence or not.